rust/src/pgsql/pgsql.rs

```diff
@@ -48,23 +48,23 @@ pub enum PgsqlEvent {
 
 #[repr(u8)]
 #[derive(Copy, Clone, PartialOrd, PartialEq, Eq, Debug)]
-pub enum PgsqlTxProgress {
+pub(crate) enum PgsqlTxProgress {
     TxInit = 0,
     TxReceived,
     TxDone,
     TxFlushedOut,
 }
```

Comment:
Clippy complained about this. It doesn't look this struct is ever used from C (just its integer values), so you can drop the stutter of the Tx prefix, since its always references like 'PgsqlTxProgress::Init`, etc.

Reply:
🙇🏽